src/Route.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getProp, isControlledProp } from './util/props';
+import withYMaps from './withYMaps';
+import * as events from './util/events';
+import applyRef from './util/ref';
+import { withParentContext } from './Context';
+
+export class Route extends React.Component {
+  constructor() {
+    super();
+    this.state = { instance: null };
+  }
+
+  componentDidMount() {
+    Route.mountObject(this.props.ymaps.route, this.props).then(instance =>
+      this.setState({ instance })
```

Comment:
This is an async process, so you'll need to check if the component is still mounted before setting the state (see [`Panorama`](https://github.com/gribnoysup/react-yandex-maps/blob/c0a399e7072a3d120a3a7d6f20c353027cbabc31/src/Panorama.js#L29) for the example of how to do this)